download.html

```diff
@@ -165,13 +165,6 @@ <h2 class="site-section__heading">Choose Your Platform</h2>
                     playtest=true %}
                 </span>
               </p>
-              <p class="text--info text--small">
```

Comment:
Also need to replace `<strong>macOS 10.13</strong>` above with `<strong>macOS 10.11</strong>`